internal/cli/images.go

```diff
@@ -53,10 +59,11 @@ func runImagesCmd(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		err = file.WriteFile(bb, fmt.Sprintf("%s/%d.jpg", output, i))
+		err = file.WriteFile(bb, fmt.Sprintf("%s/%d.jpg", output, imagesNum))
```

Comment:
I was thinking to add a `0` padding to the name, so that the images will be ordered when listed

```suggestion
		err = file.WriteFile(bb, fmt.Sprintf("%s/%03d.jpg", output, imagesNum))
```